src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends AbstractFigure {
+
+    private final double radius;
+
+    public Circle(Colors color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.PI * Math.pow(radius, 2);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Circle area: " + calculateArea() + " sq. units, radius: "
+                + radius + " units, color: " + color);
```

Comment:
One more thing here:
https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#use-name-for-getting-string-representation-of-enum-constants

Please call `name()` when referring to enum.

This applies to all usages.